pinot-common/src/main/java/org/apache/pinot/sql/parsers/parser/SqlAtTimeZone.java

```diff
@@ -18,25 +18,27 @@
  */
 package org.apache.pinot.sql.parsers.parser;
 
-import java.util.Arrays;
+import org.apache.calcite.sql.SqlBinaryOperator;
 import org.apache.calcite.sql.SqlKind;
-import org.apache.calcite.sql.SqlNode;
-import org.apache.calcite.sql.SqlSpecialOperator;
-import org.apache.calcite.sql.parser.SqlParserPos;
+import org.apache.calcite.sql.SqlOperandCountRange;
+import org.apache.calcite.sql.fun.SqlStdOperatorTable;
 import org.apache.calcite.sql.type.OperandTypes;
+import org.apache.calcite.sql.type.SqlOperandCountRanges;
 import org.apache.calcite.sql.type.SqlTypeFamily;
 import org.apache.calcite.sql.type.SqlTypeName;
 
-import static java.util.Objects.requireNonNull;
 
-
-public class SqlAtTimeZone extends SqlSpecialOperator {
+public class SqlAtTimeZone extends SqlBinaryOperator {
 
   public static final SqlAtTimeZone INSTANCE = new SqlAtTimeZone();
 
+  static {
+    SqlStdOperatorTable.instance().register(INSTANCE);
```

Comment:
see https://issues.apache.org/jira/browse/CALCITE-5302 for why this is necessary